src/validations/index.ts

```diff
@@ -0,0 +1,66 @@
+const emailRegex =
+  /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const passRegex = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{7,}$/;
+const dtRegex =
+  /^(?:(?:31(\/|-|\.)(?:0?[13578]|1[02]))\1|(?:(?:29|30)(\/|-|\.)(?:0?[13-9]|1[0-2])\2))(?:(?:1[6-9]|[2-9]\d)?\d{2})$|^(?:29(\/|-|\.)0?2\3(?:(?:(?:1[6-9]|[2-9]\d)?(?:0[48]|[2468][048]|[13579][26])|(?:(?:16|[2468][048]|[3579][26])00))))$|^(?:0?[1-9]|1\d|2[0-8])(\/|-|\.)(?:(?:0?[1-9])|(?:1[0-2]))\4(?:(?:1[6-9]|[2-9]\d)?\d{2})$/;
+const isFutureDate = (param: string) => {
+  const now = new Date();
+  const month = (now.getMonth() + 1).toString();
+  const newMonth = month.length === 1 ? '0' + month : month;
+  const day = now.getDate().toString();
+  const newDay = day.length === 1 ? '0' + day : day;
+  const year = now.getFullYear().toString();
+
+  const date = year + '/' + newMonth + '/' + newDay;
+  const newBirth = param.split('/')[2] + '/' + param.split('/')[1] + '/' + param.split('/')[0];
+
+  return Date.parse(newBirth) > Date.parse(date);
+};
+
+export const emailValidate = (param: string): boolean => {
+  if (!emailRegex.test(param)) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
Geralmente if/elses que retornam um boolean podem ser substituídos pela própria expressão do if. Até porque a expressão é booleana, né?

```suggestion
  return emailRegex.test(param);
```

Mesma coisa para os outros validadores.